### PR TITLE
Changed to boot 3 stable

### PR DESCRIPTION
**Pull request by yurichervoniy**

### CODE REVIEW

File: src/main/resources/db/changelog/properties.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+databaseChangeLog:
```
Comment (arufanov) [inline src/main/resources/db/changelog/properties.yml:1]: @yurichervoniy 
this file was crated by you or auto-generated? Looks like auto-generated. Is it correct?

**Review by arufanov — CHANGES_REQUESTED** [review]
Need discuss some issue, primary on is database schema

File: src/main/java/com/a5lab/tabr/ring/Ring.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.DynamicUpdate;
+
+import java.util.Objects;
+
+@Entity
```
[the remark this answers is not in the document]
Comment (yurichervoniy) [reply src/main/java/com/a5lab/tabr/ring/Ring.java:18]: For maven projects java doc will be generated automatically.
About  java doc - i think we need to discuss it

File: src/main/resources/db/changelog/changelog-master.xml [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+
+    <property name="now" value="now()" dbms="h2"/>
     <property name="now" value="current_timestamp" dbms="postgresql"/>
-    <property name="floatType" value="float4" dbms="postgresql"/>
-    <property name="clobType" value="longvarchar" dbms="postgresql"/>
-    <property name="blobType" value="bytea" dbms="postgresql"/>
-    <property name="uuidType" value="uuid" dbms="postgresql"/>
-    <property name="datetimeType" value="datetime" dbms="postgresql"/>
+
+    <include file="migrations/2022-11-0000-create-radar-rings-table.xml" relativeToChangelogFile="true"/>
```
[the remark this answers is not in the document]
Comment (yurichervoniy) [reply src/main/resources/db/changelog/changelog-master.xml:10]: No, i've just decided to use "-" we can use "_" as well. 
I think it's a point to discuss

File: src/main/resources/db/changelog/properties.yml (hunk 1 above)
Comment (yurichervoniy) [reply src/main/resources/db/changelog/properties.yml:1]: It's hand made =)
I've created the file using old good pattern copy-paste

File: docker/postgresql.yml [hunk 4]
```diff
@@ -5,7 +5,7 @@ services:
     image: postgres:14.6
     environment:
       - POSTGRES_USER=tabr
-      - POSTGRES_PASSWORD=
+      - POSTGRES_PASSWORD=tabr
```
Comment (arufanov) [inline docker/postgresql.yml:8]: @yurichervoniy 
I use default password: secret. I have changed applicatino.yml. It is good idea to have different name for password and user

File: src/main/resources/db/changelog/migrations/2022_11_0000_create_rings_table.xml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            <column name="last_modified_by" type="${user.type}" remarks="user who modified the record">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_modified_date" type="${datetime.type}" remarks="when record was modified"
+                    defaultValue="now()">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <!-- current Liquibase ignores checkConstraint attribute -->
+        <sql>
```
Comment (arufanov) [inline src/main/resources/db/changelog/migrations/2022_11_0000_create_rings_table.xml:44]: @yurichervoniy 
* i doubt that sql code is good for myslq, pg & sqlite.

File: src/main/resources/db/changelog/migrations/2022_11_0000_create_segments_table.xml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+            </column>
+            <column name="last_modified_by" type="${user.type}" remarks="user who modified the record">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_modified_date" type="${datetime.type}" remarks="when record was modified"
+                    defaultValue="now()">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <!-- current Liquibase ignores checkConstraint attribute -->
```
Comment (arufanov) [inline src/main/resources/db/changelog/migrations/2022_11_0000_create_segments_table.xml:36]: @yurichervoniy 
* i doubt about sqlite, mysql, pg

File: src/main/java/com/a5lab/tabr/ring/Ring.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (arufanov) [reply src/main/java/com/a5lab/tabr/ring/Ring.java]: @yurichervoniy 
* ok, thanks! it is related to checkstyle pr.

File: src/main/resources/db/changelog/migrations/2022-11-0000-create-radar-rings-table.xml [hunk 7]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <include file="../properties.yml" relativeToChangelogFile="true"/>
+
+    <changeSet id="create_radar_rings_table" author="a5lab">
+        <createTable tableName="radar_rings" remarks="Holds information about radar rings">
```
[the remark this answers is not in the document]
Comment (arufanov) [reply src/main/resources/db/changelog/migrations/2022-11-0000-create-radar-rings-table.xml]: @yurichervoniy 
* move to https://github.com/a5lab/tabr/issues/42

**Review by arufanov — APPROVED** [review]
Extra issue move to https://github.com/a5lab/tabr/issues/42

File: src/main/resources/db/changelog/data/dictionaries/fill_radar_segments.csv [hunk 8]
```diff
@@ -0,0 +1,6 @@
+id,name,description,created_by
+1,Languages,Programming languages,a5lab
```
[the remark this answers is not in the document]
Comment (arufanov) [reply src/main/resources/db/changelog/data/dictionaries/fill_radar_segments.csv]: https://github.com/a5lab/tabr/issues/42

File: src/main/resources/db/changelog/data/dictionaries/fill_radar_segments.csv [hunk 9]
```diff
@@ -0,0 +1,6 @@
...
+2,Data Management,Tools for data management,a5lab
+3,Infrastructure,Where we can deploy,a5lab
+4,Datastores,Where we can store our data,a5lab
+5,Frameworks,Frameworks we use,a5lab
```
[the remark this answers is not in the document]
Comment (arufanov) [reply src/main/resources/db/changelog/data/dictionaries/fill_radar_segments.csv]: move to https://github.com/a5lab/tabr/issues/42

File: docker/postgresql.yml (hunk 4 above)
Comment (arufanov) [reply docker/postgresql.yml:8]: move to https://github.com/a5lab/tabr/issues/42

File: src/main/resources/db/changelog/migrations/2022_11_0000_create_segments_table.xml (hunk 6 above)
Comment (arufanov) [reply src/main/resources/db/changelog/migrations/2022_11_0000_create_segments_table.xml:36]: move to https://github.com/a5lab/tabr/issues/42

**Pull request merged**